### DISCUSSION TITLE
Absolute Z offset ?

### DISCUSSION THREAD

**Discussion by bk227865** [KlipperScreen/KlipperScreen#609] [Ideas]
Is there a way to see , and modify the absolute Z offset in klipperscreen , i mean the one that is written to the printer.cfg file ?
If i notice my prints start just i tad to high i would find it easier to just manualy add 0.05 to that number without having to go through the whole physically moving the head to the -0.05 position and then saving in Z-calibration routine.  And arriving here from Marlin it's always nice to know the current actual value of that absolute number.

this one : 
#*# [bltouch]
#*# z_offset = 2.160

but then directly in Klipperscreen.

**Comment by ukmook** [comment]
Pretty sure that I adjusted the z-off last night doing some problem solving and klipper screen message asked if I wanted to make it permanent. But now I need to check again.

**Comment by alfrix** [comment]
#723
